frontend/src/metabase/sharing/components/EmailAttachmentPicker.unit.spec.js

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { render, fireEvent, screen } from "@testing-library/react";
+
+import EmailAttachmentPicker from "./EmailAttachmentPicker";
+
+describe("EmailAttachmentPicker", () => {
+  describe("when instantiated without any cards with attachments", () => {
+    let pulse;
+    let setPulse;
+    beforeEach(() => {
+      pulse = createPulse();
+      setPulse = jest.fn();
+      render(
+        <EmailAttachmentPicker
+          cards={pulse.cards}
+          pulse={pulse}
+          setPulse={setPulse}
+        />,
+      );
+    });
+
+    it("should have a Toggle that is not toggled", () => {
+      const toggle = screen.getByRole("checkbox");
+      expect(toggle).toBeInTheDocument();
+      expect(toggle.dataset.testid).toEqual("toggle");
+      expect(toggle.getAttribute("aria-checked")).toBe("false");
```

Comment:
I guess `expect(toggle).toHaveAttribute("aria-checked", "false")` is a more preferred notation. At least there is a [eslint rule](https://github.com/testing-library/eslint-plugin-jest-dom/blob/master/docs/rules/prefer-to-have-attribute.md) for that. I'd discuss adding some jest specific linters in the nearest future